source/includes/data-formats/bson.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <chrono>
+
+#include <bsoncxx/builder/basic/document.hpp>
```

Comment:
```suggestion
#include <bsoncxx/builder/basic/document.hpp>
#include <bsoncxx/builder/list.hpp>
#include <bsoncxx/builder/stream/document.hpp>
```

Add missing includes to ease compiling this file.